create_reports.py

```diff
@@ -71,7 +71,8 @@ def create_daily_report(config):
                 hour = datetime.strptime(row['Timestamp'], "%Y-%m-%d %H:%M:%S").hour
                 detections = ast.literal_eval(row['Detections'])
                 faces = [d["face_label"] for d in detections if "face_label" in d]
-                summary[hour] += (1, int(row['DetectedObjects']), int(row['ViolatingObjects']), len(faces), int(sum(faces)))
+                masks = [f for f in faces if f == 0]
```

Comment:
So `d["face_label"] == 0` when the person is using a facemask? That sounds weird

Reply:
The facemask classifier returns `class_ids`, not boolean values. This can be read in the docstring of the function.
```
"""
        Inference function sets input tensor to input image and gets the output.
        The interpreter instance provides corresponding class id output which is used for creating result
        Args:
            resized_rgb_images: Array of images with shape (no_images, img_height, img_width, channels)
        Returns:
            result: List of class id for each input image. ex: [0, 0, 1, 1, 0]
            scores: The classification confidence for each class. ex: [.99, .75, .80, 1.0]
        """
```
In the tests that I did, the value 0 is attached to the class `with mask`.